worlds/minecraft/__init__.py

```diff
@@ -44,6 +44,7 @@ def _get_mc_data(self):
             'egg_shards_required': min(self.world.egg_shards_required[self.player], self.world.egg_shards_available[self.player]),
             'egg_shards_available': self.world.egg_shards_available[self.player],
             'MC35': bool(self.world.send_defeated_mobs[self.player]),
+            'death_link': bool(self.world.death_link[self.player]),
```

Comment:
this is being overwritten in the loop at line 112 where it loops though all options and adds them to the slot data, this is causing death_link to be a "1" instead of "true"